pyproject.toml

```diff
@@ -95,22 +95,42 @@ exclude = ["3rdparty"]
 extend-select = [
     "ARG",
     "A",
+    # "ANN",
```

Comment:
Some rules can be enabled in the future to increase code quality.
"D" is quiet useful for docs